site/content/en/docs/drivers/_index.md

```diff
@@ -8,9 +8,12 @@ description: >
 aliases:
   - /docs/reference/drivers
 ---
-minikube can be deployed as a VM, a container, or bare-metal.
+Minikube can be deployed as a VM, a container, or bare-metal where a server can be used as a single tenant.
 
-To do so, we use the [Docker Machine](https://github.com/docker/machine) library to provide a consistent way to interact with different environments. Here is what's supported:
+Docker and Podman are container engines with almost similar operating commands. Podman interacts directly with the container, whereas Docker uses daemon for this.
```

Comment:
s/almost/very/g

I also don't think it's important to mention podman here.

Reply:
This could say something like

```
minikube can run as a container and currently supports running on docker (recommended) and podman.
```